src/app/collection/pages/collection-ncyte/resource/resource.component.html

```diff
@@ -0,0 +1,35 @@
+
+    <div class="section-resource">
+        <div class="resource-info-1">
+            <img class = "logo-tech-ncyte" src="../../../../assets/images/collections/ncyte.png" alt="">
+            <p class="desc-ncyte">
+                The National Cybersecurity Training & Education (NCyTE) Center advances cybersecurity education in the U.S. 
+                by investing in technological innovation, resources, professional development and tools to support faculty, 
+                community colleges and the workforce pipeline of tomorrow.
+                Details at: <aria-label href=" http://ncyte.net">www.ncyte.net</aria-label>
```

Comment:
there isn't an aria-label tag, it should be an a tag with an aria-label attribute